chingu-voyages/v51-tier3-team-33
# Gary/feature/update group

# Description
This commit includes the functionality to update the info a group. - This is built on top of update-delete-expenses.

## Related to the following Jira/Trello issue 
N/A

## Main changes explained:
- Created updateGroup api to handle updating group info.

## How to test:
1. check into current branch
2. do `npm install` and `npm run dev` to run this PR locally
3. Open postman and make a PUT request to http://localhost:3000/api/groups/*enter group id here * /updateGroup, the body of the request should contain the name, description, and budget.
4. The request should return the updated group with the new info. If so, you're good to go.
